tests/integration/helpers.py

```diff
@@ -0,0 +1,110 @@
+import yaml
+from workload import Prometheus
+
+
+async def unit_address(ops_test, app_name, unit_num):
+    """Find unit address for any application.
+
+    Args:
+        ops_test: pytest-operator plugin
+        app_name: string name of application
+        unit_num: integer number of a juju unit
+
+    Returns:
+        unit address as a string
+    """
+    status = await ops_test.model.get_status()
+    return status["applications"][app_name]["units"][f"{app_name}/{unit_num}"]["address"]
+
+
+async def check_prometheus_is_ready(ops_test, app_name, unit_num):
```

Comment:
```suggestion
async def check_prometheus_is_ready(ops_test: OpsTest, app_name: str, unit_num: int) -> bool:
```

Reply:
(I won't add suggestions or all, but seems like adding type annotations to helper methods is a good idea throughout)

Reply:
Ack. Will add type hints.